lib/http_fuzzer/engine.py

```diff
@@ -41,6 +41,16 @@ def user_agents_list():
         "Mozilla/5.0 (compatible; YandexBot/3.0; +http://yandex.com/bots)",
         "Mozilla/5.0 (compatible; bingbot/2.0; +http://www.bing.com/bingbot.htm)",
         "msnbot/1.1 (+http://search.msn.com/msnbot.htm)"
```

Comment:
`"msnbot/1.1 (+http://search.msn.com/msnbot.htm)",` instead of `"msnbot/1.1 (+http://search.msn.com/msnbot.htm)"`. The comma is missing, please test your changes before the pull request.